relay-config/src/config.rs

```diff
@@ -502,14 +502,6 @@ struct Metrics {
     /// For example, a value of `0.3` means that only 30% of the emitted metrics will be sent.
     /// Defaults to `1.0` (100%).
     sample_rate: f32,
-    /// Code locations expiry in seconds.
-    ///
-    /// Defaults to 15 days.
-    meta_locations_expiry: u64,
-    /// Maximum amount of code locations to store per metric.
-    ///
-    /// Defaults to 5.
-    meta_locations_max: usize,
```

Comment:
Have we considered self-hosted users (or us) that may be using this config here?

I think doing this is fine as long as we document it properly in the changelog.

Reply:
Nobody is using this config here, we just added it and it is still a very experimental feature, but this is a valid concern for the partitions and batch size settings, we will also have to update these in our prod configs before/after merging this.